### PR TITLE
Scaling graphite

### PR DESCRIPTION
**Pull request by gjmulder**
Added new post and corresponding author entries for myself and J.D.

### CODE REVIEW

**Comment by WheresAlice** [comment]
I'd be interested in whether you considered any sort of virtualization on this new hardware as a way of making management easier. And whether virtualization would really have made a big difference in performance.

**Comment by gjmulder** [comment]
No, virtualisation was never considered. Storage was intended to be local and fast, which means that we'd have payed a small price in virtualisation latency overhead. Local storage also eliminated the possibility of VMotioning VMs. We could of course simply bought a dedicated SAN for Graphite storage and continued to use VMs, but that would have added considerably to the cost of the project.

File: _data/authors.yml [hunk 1]
```diff
@@ -181,3 +181,15 @@ adam_blanchard:
...
+    name: John Denholm
+    Lead role: Lead Platform Engineer
+    tribe: Bet
+    bio: Longtime Skybet engineer, generally in charge of all things Graphite. Likes cats and whiskey.
+
+gary_mulder:
+    name: Gary Mulder
+    role: Site Reliability Engineer
+    tribe: Bet
+    bio: After 10 years as a Sytems Administrator, now specialising in Site Reliability and Performance Analysis. Likes making fast computers run faster. And maximum-pain chilli peppers.
```
Comment (medains) [inline _data/authors.yml]: Typo in your bio.

**Comment by medains** [comment]
Nice article - it's maybe worth linking to the Graphite homepage, and as an aside - do we want to get SB&G added to the list of "who is using graphite" in their docs?

**Comment by adamhepton** [comment]
The authors are missing some keys, and also, you need to have images representing you as people ;)
```
author_name:
    name: Author Name
    image: author_name.png
    role: Author's Role / Squad
    tribe: Author's Tribe
    startDate: 01-05-2015
    bio: Some information about the author, can be as long as you like, but should be written in proper English.
    interests: Comma, Separated, Interests, List
    twitter: optional_twitter_handle
    github: optional_github_account
```

**Comment by ls12styler** [comment]
@adamhepton @medains: Not sure what to do with this. It's a good post and would be remiss to let it disappear. Shall we give it to JD to rebase and fix up?

**Comment by adamhepton** [comment]
It is a good post and I think we should definitely publish it.  I'll add it to my list tonight - I'll fork from Gary's branch (he's since left us) and change up the authors stuff, add an image and a link to Graphite, and drop a new PR.

**Comment by adamhepton** [comment]
Closed, this is going to be reopened as a rebase with the missing bits and posted onto site on Friday 13th January 2017.  Hello Gary, we miss you!

**Comment by gjmulder** [comment]
:-)

**Pull request closed without merging**